scRNA-seq/02-filtering_scRNA.Rmd

```diff
@@ -341,8 +349,7 @@ ggplot(mapping = aes(x = mito_fraction)) +
   theme_classic()
 ```
 Here, we want to keep cells with a low fraction of reads corresponding to mitochondrial genes and remove any cells with a high mitochondrial fraction.
-
-
+Again, it's important to take this step even if you have performed some initial filtering, since mapping software like `salmon`'s quality control does not consider mitochondrial read percentages.
```

Comment:
I don't know we need to repeat this here, but if we do:
```suggestion
Again, it's important to take this step even if you are started with filtered data, since mapping software like `salmon alevin` and Cell Ranger do not usually consider mitochondrial read percentages when filtering.

```